introduction.md

```diff
@@ -0,0 +1 @@
+Hello everyone.  This is my formal introduction to the dojo. My name is Chad, username: chadjguidry.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.